packages/eds-core-react/src/components/Radio/Radio.docs.mdx

```diff
@@ -0,0 +1,70 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Radio } from '.'
+
+# Radio
+
+Allow users to select only one option from a list.
+
+<Links 
+    ariaUrl="https://www.w3.org/WAI/ARIA/apg/patterns/radiobutton/"
+    figmaUrl="https://www.figma.com/file/0bGXR2sCwMVSDNyyzu5BXrO5/UI%E2%80%94User-Interface?node-id=18442%3A38197"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/028476-selection-controls/b/86826a"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Radio/Radio.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+/>
+
+<Story id="inputs-selection-controls-radio--introduction" />
+<PropsTable />
+
+## Usage
+
+**Selection controls**
+
+<ul>
+    <li>Allow users to select options, make decisions and set preferences.</li>
+    <li>Are to be visible and understandable at a quick glance.</li>
+    <li>Are always accompanied with a clear label.</li>
+</ul>
+
+**Radio**
+
+<ul>
+    <li>Parent must have <code>position: relative</code></li>
```

Comment:
Why? This seems a bit restrictive and we don't have this in our examples?

Reply:
This one was mentioned in the documentation

Reply:
Maybe it was a requirement in a much older version of the code? It is definetly not needed in the current version 🤷‍♂️

Reply:
I took it away.